Videos/Actions/Actions/Actions.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Syncfusion.Pdf.Net.Core" Version="31.1.20" />
```

Comment:
@deepika134 , Can you remove the version and mention it as * so that it will automatically install the latest version of our NuGet packages.

<PackageReference Include="Syncfusion.Pdf.Net.Core" Version="*" />